### ISSUE TITLE
Periodically perform check redistribution

### ISSUE THREAD

**Issue by dselans** [9corp/9volt#12]
Perform periodic check redistribution if one of the member nodes has disproportionately more/less checks than other members.

**Comment by jessedearing** [comment]
Would this be likely to occur?

I always thought that if the cluster changed (members added or deleted) then the checks get rebalanced.

**Comment by dselans** [comment]
You're right and that functionality already works (which is cool++).

I don't remember the exact case I thought of when I wrote this issue, but I *just* created another case where this could happen.

There is a fairly complex race that is pretty unlikely to happen, but here it is anyway:

1. When a node is first started
2. And a new check is added to the configuration (via say `curl`)
3. But the director has not yet had enough time to determine check distribution (it runs in the background on an interval to figure out how many checks each member has)
4. The director *will* return itself as the node where the new check should be pinned to

I deliberated on this for a bit - it could be changed to "pick random node", but settled on director so that when/if we run into this behavior, it is a bit easier to troubleshoot.

I'll see if I can figure out why I thought this needed a separate issue in the first place though.
